openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -103,6 +103,18 @@
             });
 
             sectionOneFieldViews = [
+                new FieldsView.DateFieldView({
```

Comment:
Just toss this after the Location and Language (if I remember the wireframes correctly).